lib/rules/declaration-property-value-no-unknown/index.mjs

```diff
@@ -64,16 +64,30 @@ const rule = (primary, secondaryOptions) => {
 		const propertiesSyntax = {
 			overflow: '| overlay', // csstree/csstree#248
 			width: '| min-intrinsic | -moz-min-content | -moz-available | -webkit-fill-available', // csstree/csstree#242
+			'anchor-name': 'none | <custom-property-name>#',
```

Comment:
`<dashed-ident>` is not exposed.

see https://github.com/csstree/csstree/blob/ba6dfd8bb0e33055c05f13803d04825d98dd2d8d/lib/lexer/generic.js#L225-L227